linux/user-and-file-management/user-permissions/file-ownership.md

```diff
@@ -42,33 +28,26 @@ On Linux, every file and directory is given both a user owner and a group owner.
 Here is some example output:
 
 ```bash
-$ ls -ld /var/log
+ls -ld /var/log
 drwxrwxr-x 10 root syslog
     4096 Jun 13 06:25 /var/log
-$ ls -l /var/log/syslog
+ls -l /var/log/syslog
```

Comment:
At this point i'd take advantage of scrollable snippets and make sure the output is not split over multiple lines

Reply:
that is true, but I still think that long lines should be split. it might not be clear to all users that snippets are scrollable.

also one long line might be harder to follow due to the back and forth scrolling